cmake/FindFFmpeg.cmake

```diff
@@ -76,15 +77,15 @@ endmacro()
 # include directories.
 #
 macro(find_component _component _pkgconfig _library _header)
-  if(NOT WIN32)
+    if(NOT WIN32)
     # use pkg-config to get the directories and then use these values
     # in the FIND_PATH() and FIND_LIBRARY() calls
     find_package(PkgConfig)
 
     if(PKG_CONFIG_FOUND)
       pkg_check_modules(PC_${_component} QUIET ${_pkgconfig})
     endif()
-  endif(NOT WIN32)
+  endif(WIN32)
```

Comment:
`NOT WIN32` as said by the various checkers

Reply:
yepp, should be fixed now.